tox.ini

```diff
@@ -7,7 +7,9 @@
 envlist = py2,py3,pypy,cover
 
 [testenv]
-commands = pytest
+passenv =
+    FIREBASE_DATABASE_EMULATOR_HOST
```

Comment:
What does this do exactly? We use this tox config during our releases. Will it be affected in any way?

Reply:
tox by default filters out most environment variables when running subprocesses like `tox`. This line basically tells tox that if `FIREBASE_DATABASE_EMULATOR_HOST` is set (e.g. `FIREBASE_DATABASE_EMULATOR_HOST=localhost:9000 tox`) then tox should pass along this env to pytest. This won't affect releases since we do not set `FIREBASE_DATABASE_EMULATOR_HOST` during releases.